package.json

```diff
@@ -78,6 +79,7 @@
     "reactifex": "1.1.1",
     "redux-mock-store": "^1.3.0",
     "sass-loader": "^6.0.7",
+    "semantic-release": "^15.1.7",
```

Comment:
Probably want to add `travis-deploy-once` as per [the `v12.0.0` release recommendation](https://github.com/semantic-release/semantic-release/releases/tag/v12.0.0)

![image](https://user-images.githubusercontent.com/8136030/39013712-3814cb10-43e6-11e8-93a6-680fa5c76698.png)